### DISCUSSION TITLE
Не находит главы манги

### DISCUSSION THREAD

**Discussion by uchihaamaya** [lirix360/ReadmangaGrabber#84] [Q&A] [answered]
Уже пару дней не находит главы любой манги. Например, https://mintmanga.live/emocuhegi__plod_preispodnei . Пишет: "
Глав не найдено. Проверьте правильность ввода адреса манги." и так с любой мангой. Авторизация пройдена (зеленая галочка). Кто-нибудь еще сталкивался?

**Comment by lirix360** [comment, marked as answer]
Исправлено в [версии 2.1.7](https://github.com/lirix360/ReadmangaGrabber/releases/latest)
